TTS/tts/layers/xtts/tokenizer.py

```diff
@@ -476,6 +476,28 @@ def __init__(self, vocab_file=None, preprocess=None):
                 self.preprocess = preprocess
 
             self.tokenizer = Tokenizer.from_file(vocab_file)
+    
+    def check_input_length(self, txt, lang):
+        char_limits = {
```

Comment:
For performance, this should probably be a global constant so it isn't recomputed for each call of this function?

Reply:
Good point, thx